src/app.js

```diff
@@ -119,8 +117,29 @@ function createApp (config, ledgers, backend, routeBuilder, routeBroadcaster, ro
   require('./services/auth')(passport, config)
 
   // Logger
-  koaApp.use(logger())
-  koaApp.use(errorHandler({log: log('error-handler')}))
+  koaApp.use(logger(log.create('koa')))
+  koaApp.use(logger.requestIdContext())
+
+  const isTrace = log.trace()
+  koaApp.use(logger.requestLogger({
+    updateRequestLogFields: function (fields) {
+      return {
+        headers: this.req.headers,
+        body: isTrace ? this.body : undefined,
+        query: this.query
+      }
+    },
+    updateResponseLogFields: function (fields) {
+      return {
+        duration: fields.duration,
+        status: this.status,
+        headers: this.headers,
+        body: isTrace ? this.body : undefined
+      }
+    }
+  }))
+  koaApp.use(errorHandler({log: log.create('error-handler')}))
+  koaApp.on('error', function () {})
```

Comment:
If we were showing them before, then I think we should keep showing them because we only show stack traces when there is a problem that should never happen. For things that should never happen I think its best to show it everywhere because when you are developing it will waste time if the error is in the log and you aren't looking at the log at that moment.